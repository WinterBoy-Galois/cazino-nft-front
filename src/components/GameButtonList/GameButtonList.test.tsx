import React from 'react';
import { render } from '@testing-library/react';
import GameButtons from './GameButtonList';
import { LocationProvider } from '@reach/router';

describe('GameButtonList', () => {
  it('should match snapshot', () => {
    // Arrange

    // Act
    const container = render(
      <LocationProvider>
        <GameButtons />
      </LocationProvider>
    );

    // Assert
    expect(container).toMatchSnapshot();
  });
});
