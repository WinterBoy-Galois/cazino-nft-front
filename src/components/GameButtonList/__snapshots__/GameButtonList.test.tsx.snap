// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameButtonList should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row"
      >
        <div
          class="col-12 col-md-6 col-xl-6 "
        >
          <div
            class="container container"
          >
            <h2
              class="headline"
            >
              Goals
            </h2>
            <span
              class="subline"
            >
              Play
            </span>
            <div
              class="highlight"
            >
              <img
                alt="Ball"
                class="highlight__animated"
                src="goal-ball.svg"
              />
              <img
                alt="Field"
                class="highlight__static"
                src="goal-field-home-button.svg"
              />
            </div>
            <img
              alt="Goal Keeper"
              class="character "
              src="goal-character-home-button.svg"
            />
          </div>
        </div>
        <div
          class="col-12 col-md-6 col-xl-6 "
        >
          <div
            class="container container"
          >
            <h2
              class="headline"
            >
              Mines
            </h2>
            <span
              class="subline"
            >
              Play
            </span>
            <div
              class="highlight"
            >
              <img
                alt="Bomb"
                class="highlight__animated"
                src="mines-bomb.svg"
              />
              <img
                alt="Shadow"
                class="highlight__static"
                src="mines-shadow-home-button.svg"
              />
            </div>
            <img
              alt="Soldier"
              class="character "
              src="mines-character-home-button.svg"
            />
          </div>
        </div>
        <div
          class="col-12 col-md-6 col-xl-6 "
        >
          <div
            class="container container"
          >
            <h2
              class="headline"
            >
              Clams
            </h2>
            <span
              class="subline"
            >
              Play
            </span>
            <div
              class="highlight"
            >
              <img
                alt="Pearl"
                class="highlight__animated"
                src="clam-pearl.svg"
              />
            </div>
            <img
              alt="Clam"
              class="character "
              src="clams-character-home-button.svg"
            />
          </div>
        </div>
        <div
          class="col-12 col-md-6 col-xl-6 "
        >
          <div
            class="container container"
          >
            <h2
              class="headline"
            >
              Dice
            </h2>
            <span
              class="subline"
            >
              Play
            </span>
            <div
              class="highlight"
            >
              <img
                alt="Bomb"
                class="highlight__animated"
                src="dices-dices.svg"
              />
              <img
                alt="Shadow"
                class="highlight__static"
                src="dice-shadow-button.svg"
              />
            </div>
            <img
              alt="Soldier"
              class="character "
              src="dice-character-home-button.svg"
            />
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row"
    >
      <div
        class="col-12 col-md-6 col-xl-6 "
      >
        <div
          class="container container"
        >
          <h2
            class="headline"
          >
            Goals
          </h2>
          <span
            class="subline"
          >
            Play
          </span>
          <div
            class="highlight"
          >
            <img
              alt="Ball"
              class="highlight__animated"
              src="goal-ball.svg"
            />
            <img
              alt="Field"
              class="highlight__static"
              src="goal-field-home-button.svg"
            />
          </div>
          <img
            alt="Goal Keeper"
            class="character "
            src="goal-character-home-button.svg"
          />
        </div>
      </div>
      <div
        class="col-12 col-md-6 col-xl-6 "
      >
        <div
          class="container container"
        >
          <h2
            class="headline"
          >
            Mines
          </h2>
          <span
            class="subline"
          >
            Play
          </span>
          <div
            class="highlight"
          >
            <img
              alt="Bomb"
              class="highlight__animated"
              src="mines-bomb.svg"
            />
            <img
              alt="Shadow"
              class="highlight__static"
              src="mines-shadow-home-button.svg"
            />
          </div>
          <img
            alt="Soldier"
            class="character "
            src="mines-character-home-button.svg"
          />
        </div>
      </div>
      <div
        class="col-12 col-md-6 col-xl-6 "
      >
        <div
          class="container container"
        >
          <h2
            class="headline"
          >
            Clams
          </h2>
          <span
            class="subline"
          >
            Play
          </span>
          <div
            class="highlight"
          >
            <img
              alt="Pearl"
              class="highlight__animated"
              src="clam-pearl.svg"
            />
          </div>
          <img
            alt="Clam"
            class="character "
            src="clams-character-home-button.svg"
          />
        </div>
      </div>
      <div
        class="col-12 col-md-6 col-xl-6 "
      >
        <div
          class="container container"
        >
          <h2
            class="headline"
          >
            Dice
          </h2>
          <span
            class="subline"
          >
            Play
          </span>
          <div
            class="highlight"
          >
            <img
              alt="Bomb"
              class="highlight__animated"
              src="dices-dices.svg"
            />
            <img
              alt="Shadow"
              class="highlight__static"
              src="dice-shadow-button.svg"
            />
          </div>
          <img
            alt="Soldier"
            class="character "
            src="dice-character-home-button.svg"
          />
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
